src/docs/PEO/PEOUserGuide.adoc

```diff
@@ -1,87 +1,87 @@
 :docinfo:
 :toc: 
-:toc-title: PCW User Guide
-= ProActive Cloud Watch (PCW)
+:toc-title: PEO User Guide
+= ProActive Event Orchestration (PEO)
 include::../common-settings.adoc[]
 include::../all-doc-links.adoc[]
 
-== Main concepts of PCW
+== Main concepts of PEO
 
-*ProActive Cloud Watch (PCW)* monitors a system to detect complex events, and triggers actions when activated rules are fired. +
-PCW is configured with *rules* to define the metrics to monitor, actions to be performed, and conditions to be met to trigger them. +
+*ProActive Event Orchestration (PEO)* monitors a system to detect complex events, and triggers actions when activated rules are fired. +
+PEO is configured with *rules* to define the metrics to monitor, actions to be performed, and conditions to be met to trigger them. +
 
-ProActive Cloud Watch is organized as a Catalog of rules (which can be activated or deactivated) that will automatically trigger user-specified actions if defined conditions are fulfilled. +
-For example, PCW is able to monitor some computing resources (ProActive Nodes) and deploy new resources (scale up) when the monitored resources are overloaded. Another use case is to notify users with alerts about some deteriorated resources. +
-In general PCW allows to trigger any action by submitting a workflow to the ProActive Scheduler.
+ProActive Event Orchestration is organized as a Catalog of rules (which can be activated or deactivated) that will automatically trigger user-specified actions if defined conditions are fulfilled. +
+For example, PEO is able to monitor some computing resources (ProActive Nodes) and deploy new resources (scale up) when the monitored resources are overloaded. Another use case is to notify users with alerts about some deteriorated resources. +
+In general PEO allows to trigger any action by submitting a workflow to the ProActive Scheduler.
 
 *Definitions:*
 
 Rule:: - defines a monitoring behavior, triggering actions according to some predefined conditions. Rules generally use a template for certain monitoring use cases, configurable using specific variables.
-Rule Instance:: - instantiated rule with concrete set of variables. One rule can have several rule Instances with different variable values. The rule instance can be activated/deactivated. PCW service will monitor metrics defined inside the rule only for activated instances.
+Rule Instance:: - instantiated rule with concrete set of variables. One rule can have several rule Instances with different variable values. The rule instance can be activated/deactivated. Proactive Cloud Watch service will monitor metrics defined inside the rule only for activated instances.
```

Comment:
Proactive Event Orchestration

Reply:
Nope, here we are describing the backend service. Event orchestration is used for the portal.